src/filters/reframer.c

```diff
@@ -241,7 +241,7 @@ GF_Err reframer_configure_pid(GF_Filter *filter, GF_FilterPid *pid, Bool is_remo
 	for (i=0; i<ctx->saps.nb_items; i++) {
 		switch (ctx->saps.vals[i]) {
 		case 1: ctx->filter_sap1 = GF_TRUE;
-			gf_filter_pid_set_property(st->opid, GF_PROP_PID_INTRA_ONLY, &PROP_BOOL(GF_TRUE));
+			gf_filter_pid_set_property(st->opid, GF_PROP_PID_HAS_SYNC, &PROP_BOOL(GF_FALSE)); //false: all samples are sync
 			break;
 		case 2: ctx->filter_sap2 = GF_TRUE; break;
```

Comment:
could you also add that for SAP2 (in case we find such a packet one of these days) - it's technically still closed gop.
The rest looks good to me thanks !

Reply:
addressed